pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/executor/OpChainSchedulerService.java

```diff
@@ -146,11 +146,6 @@ public final void register(OpChain operatorChain) {
         operatorChain,
         operatorChain.getReceivingMailbox(),
         _scheduler.size());
-
-    // we want to track the time that it takes from registering
-    // an operator chain to when it completes, so make sure to
-    // start the timer here
-    operatorChain.getStats().startExecutionTimer();
```

Comment:
changing this will only record the execution time but not the first register-to-executing delay yes?

Reply:
no. executing time shouldn't include register to execute delay. and the delay is tracked in op chain queue time.